velox/docs/functions/spark/decimal.rst

```diff
@@ -2,49 +2,115 @@
 Decimal Operators
 =================
 
-When calculating the result precision and scale of arithmetic operators,
-the formulas follow Hive which is based on the SQL standard and MS SQL:
+The result precision and scale computation of arithmetic operators contains two stages.
+First stage computes precision and scale using formulas based on the SQL standard, and Hive when allow-precision-loss is true.
```

Comment:
> based on the SQL standard, and Hive when allow-precision-loss is true.

This is a bit confusing. SQL standard is different from Hive, no? Are you saying that we match SQL standard when allow-precision-loss is false and HiveQL when allow-precision-loss is true?

Reply:
No, the SQL standard is same with Hive, we match both of them when allow-precision-loss is true.
Spark config description
```
val DECIMAL_OPERATIONS_ALLOW_PREC_LOSS =
    buildConf("spark.sql.decimalOperations.allowPrecisionLoss")
      .internal()
      .doc("When true (default), establishing the result type of an arithmetic operation " +
        "happens according to Hive behavior and SQL ANSI 2011 specification, i.e. rounding the " +
        "decimal part of the result if an exact representation is not possible. Otherwise, NULL " +
        "is returned in those cases, as previously.")
      .version("2.3.1")
      .booleanConf
      .createWithDefault(true)
```